src/theme/DefaultTheme.tsx

```diff
@@ -223,9 +209,9 @@ function ConditionEditor({
           </tr>
         </thead>
         <tbody>
-          {children.map(pair => {
+          {children.map((pair, i) => {
             return (
-              <tr>
+              <tr key={i}>
```

Comment:
I don't believe this will work with re-ordering of ASTs: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Yeh, I feel like this PR and branch can probably just be deleted. React complains a lot about keys on children for the editor and we probably need a new approach to it. Plus, with the math changes this cleanup probably needs to be done again and at another stage.